src/index.ts

```diff
@@ -57,7 +57,7 @@ export class Duffel {
     this.aircraft = new Aircraft(this.client)
     this.airlines = new Airlines(this.client)
     this.airports = new Airports(this.client)
-    this.links = new Links(this.client)
+    this.links = new Sessions(this.client)
```

Comment:
Context on this: https://duffel.slack.com/archives/C04BCM0H42W/p1676043687668789?thread_ts=1676043331.705249&cid=C04BCM0H42W